src/components/Venues/VenueModal.tsx

```
@@ -75,56 +75,105 @@ const VenueModal = ({
+          onHide();
+        }
+      } catch (error) {
+        console.error('Mutation error:', error);
+        errorHandler(t, error);
+      }
+      return;
+    }
+
+    // Validate capacity
+    const capacityNum = parseInt(formState.capacity, 10);
     if (isNaN(capacityNum) || capacityNum <= 0) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)